src/views/catalog/CreateFromInstanceTypes/components/CreateVMFooter/CreateVMFooter.tsx

```diff
@@ -118,6 +119,13 @@ const CreateVMFooter: FC = () => {
           autoUpdateEnabled,
         ),
       );
+      vmSignal.value = generateVM(
```

Comment:
Toward the end of the video the video creator said that the recommended way to use signals in components is to inject them as props in order to make testing easier. Is there a reason why you didn't do that here and in the other place the signal is used below?

Reply:
What do u mean inject them as prop? 
1. for testing purpose, while we don't do unit testing, there isn't any difference as e2e will work out of the box.
2. this line is we we load the the initial state to vmSignal

Reply:
Seems like the only advantage is unit testing. I prefer much on direct consuming as a much bigger advantage of avoiding prop drilling abusing. Which is the biggest advantage of store. In any case if in the future unit testing will be added one mock file can solve store issue to all related test.